src/chrome/content/rules/kiro7.com.xml

```diff
@@ -0,0 +1,20 @@
+<!--
+	Invalid certificate:
+		a135.kiro7.com
+		amp.kiro7.com
+		click.e.kiro7.com
+		image.e.kiro7.com
+		li.kiro7.com
+		sa135.kiro7.com
```

Comment:
The page tries to load some scripts from there. I don't know if they are used somewhere. Do we care?

Reply:
Hum, I don't see this.

Reply:
@jeremyn could you double check?

Reply:
I'm not sure what I'm checking. https://sa135.kiro7.com has an invalid certificate, and I don't see any network requests to `sa135.kiro7.com` from https://www.kiro7.com/.